JapaneseFilter/sections/specific.txt

```diff
@@ -146,6 +146,7 @@ freesoft-100.com##.related-article
 freesoft-100.com##div[class^="ad_"]
 freesoft-100.com##div[style="font-size:15px;margin:30px 0 30px 18px"]
 sekisei.click##.ad-bottom-right
+mankitu-blog.seesaa.net##.article__content > table
```

Comment:
Not an ad. This link is a part of the article in which the user can buy or not the reviewed product.